src/components/player-form.jsx

```diff
@@ -10,7 +10,8 @@ module.exports = React.createClass( {
         name: null,
         image: null,
         league: null,
-        score: null
+        score: null,
+        streak: null
```

Comment:
this isn't need here as it's just for the validations and since it's not user input we can be pretty sure of the value.